ktor-io/common/src/io/ktor/utils/io/Deprecation.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2014-2023 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.utils.io
+
+internal const val IO_DEPRECATION_MESSAGE = """
+    We're migrating to the new kotlinx-io library.
+    This declaration is deprecated and will be removed in Ktor 4.0.0
```

Comment:
Should we explain why there is no replacement yet and what people should do if it's in our public API?

Reply:
This API will be dropped without replacement